### DISCUSSION TITLE
Question about some of the command line switches and output

### DISCUSSION THREAD

**Discussion by matthewkleinmann** [TencentARC/GFPGAN#173] [Q&A]
First, thank you for the incredible program.  I have been having a ton of fun with it, and your set up instructions are great too, it was one of the easier things I have got running on Windows with no GPU.

I have a few questions, these are not issues as I have it up and going but I am curious.

1) For the command line switch -bg_upsampler, what are the options besides the default?

2) For the command line switch -bg_tile , can you elaborate a bit more on what it does?  I changed it to 4 on and old photo negative I shot on tri x film so it had considerable grain in it and it seemed to smooth it out more.  I did a second run at 10 and it seemed better yet.  It would not take 1, but it did swallow 2 and it is still going.  I think this test is going to day a couple of days.  Fingers crossed we don't lose power..  But anything you can add to how this switch effects the finished image would be appreciated.  

And last, on the output when it is showing the tile statistics, occasionally one of them will have an underscore or tick next to it like there was something unusual about that one.  Can you shed some light on what that is about?

Thank you!

**Comment by cristiano884** [comment]
como faço pra ter a restalrações de fotos?

**Comment by dilzinarhubey-cmyk** [comment]
![20251217160953.png](https://github.com/user-attachments/assets/a95fd9af-75b3-495c-9bcf-287855514c72)

**Comment by dilzinarhubey-cmyk** [comment]
![20251217160953.png](https://github.com/user-attac![20251217160953.png](https://github.com/user-attachments/assets/e8e4babe-906e-4f4a-a82e-1668070cfe1a)

hments/assets/4cf52ebb-4781-4324-9436-c5518de64950)

**Reply by dilzinarhubey-cmyk** [reply]
![Screenshot_2025-12-15-14-51-45-275_photoframe.lovecollage.truelove.loveframes.jpg](https://github.com/user-attachments/assets/8732974b-17d0-45ff-a89a-0664dccfcf67)

**Comment by dilzinarhubey-cmyk** [comment]
![20251217160953.png](https://github.com/user-attachments/assets/a5e694b1-ffd9-4824-8fb0-51b9a1703bd4)
